src/aap_eda/core/models/activation.py

```diff
@@ -102,6 +102,16 @@ class Meta:
         null=True,
         default=None,
     )
+    credentials = models.ManyToManyField(
+        "Credential", related_name="activations", default=None
```

Comment:
Now that we have a devel branch, we don't need the fanout branch anymore. I feel it would be better if the we merge fanout branch here before, because the credential should have a relation with the source too. Thoughts? @hsong-rh @mkanoor

Reply:
can we focus on vault support in this devel branch now and merge fanout later?

Reply:
Fanout branch was created to not create conflicts in main same as devel. The devel branch is for all the changes for the next release not only for vault. Unless we decide that fanout is not ready to be merged as it is, I think we should merge them. All the work for the next release should start from and be merged on the same branch.

Reply:
Is the fanout modeling going to change do we have to wait for it to be merged before we start the vault work. if the fanout model is still in a state of flux we should let this one be separate.